openmicroscopy/ome-documentation
# Light admin doc follow-up

This is a follow-up of https://github.com/openmicroscopy/ome-documentation/pull/1617. As the features develop, this PR is trying to adjust the doc correspondingly.


TODO:
Write a doc explaining precisely what combinations of privileges are hidden under the WebUI chackboxes and link it to this doc and vice versa.

The doc of Web UI checkboxes mapping was added in the last commit here.
This doc maps the checkboxes in the OMERO.web admin UI for the creation of the light admins to the back-end permissions, as defined by @mtbc .

To test:

  - check logic, spelling, etc.
  - check crosslinking with other docs
  - Also see OmeroWebGateway.setConfigRoles.


@joshmoore @jburel @will-moore @mtbc
